docs/examples/distributed/index.rst

```diff
@@ -2,6 +2,11 @@
 Distributed Training
 ********************
 
-.. include:: /examples/distributed/001_single_gpu/_index.rst
-.. include:: /examples/distributed/002_multi_gpu/_index.rst
-.. include:: /examples/distributed/003_multi_node/_index.rst
+.. toctree::
+    :maxdepth: 1
+    :glob:
+    :numbered:
+
+    single_gpu/index
+    multi_gpu/index
+    multi_node/index
```

Comment:
```suggestion
    */index
```

Reply:
The ordering of the examples is important here (they are numbered), so I can't use the glob this way.

Reply:
ah yes you're right